### DISCUSSION TITLE
Sample composition reporting

### DISCUSSION THREAD

**Discussion by NRayya** [NFDI4Chem/nmrxiv#863] [Features]
Due to the impact of impurities in the samples, especially at the biological level, we would like to encourage nmrXiv users to report the degree of the compound purity in the sample along with all known impurities. 

We would like to enable users to report the composition as weight percentage, with "N/A" if the user doesn't provide a value and with decimal values. 

What we would like to discuss:
1. Is it enough to represent the composition as percentage of weight? or are other concentration representations used usually?
2. How many decimal places should be provided to cover the impurities?
3. In many cases, the user knows the main compound in the sample without knowing its percentage exactly, we would still be interested in highlighting this compound among the reported impurities. Would a flag of "extracted compound" or "impurity" suffice? 
4. How should we ask the user to consider a compound as an impurity? based on the concentration being less than a certain threshold? based on what the user targeted to keep during the purification? else?

**Comment by tilfischer** [comment]
Dear Noura, 

I will try to give an answer, as nobody replied so far.

1. Firstly, we might need to distinguish between a value given for purity and a values to describe the composition. **Purity** is usually reported as weight percentage, with respect to the main compound. We had a brief discussion on this somewhere else before. If a spectrum looks clean i.e. there are only the signals of the main compound plus the residual proton signal of the solvent, people tend to say "its pure!". However, strictly speaking this can only be told if quantitative NMR is applied e.g. by addition of an internal standard. For the **composition**, one might need to remember that the intensity I of a signal is proportional to the amount of substance n and the number of protons N the signal corresponds to, I ~ n*N. Hence, an information easily obtained is the ratio of e.g. E/Z-Isomers or diastereomers, as these have the same molar mass (cf. n = m / M, with amount of substance n, mass m and molar mass M). Let's say the spectrum locks clean and there are only two isomers present with the same molar mass, then people might easily provide the composition as weight percentage. However, this is not possibly of the molar mass of one component is unknown. Then, people might provide the purity. In short: Such information should only be provided if there is enough evidence. For the purity, people might argue that the NMR looks clean. Due to this one might ask the submitters to tell where the provided purity value comes from i.e. if it is estimated based on the qualitative NMR (spectrum looks clean) or if this was actually determined by quantitative NMR e.g. by addition of an internal standard.
2. As NMR spectroscopy is, unfortunately, not a very sensitive technique, I would not allow any digits after the decimal point.
3. Yes, in many cases users do not know the exact percentage (do you mean purity here?) of purity/composition, as this was not quantified using e.g. an internal standard. Very frequently, a sample is added to a NMR tube, dissolved, measured and the spectrum looks clean, but researchers do not invest more time (and money) in additionally doing e.g. a measurement with an internal standard. Flags you mentioned could be handy here. One option could be "Impurity", but I would not suggest to use "extracted compound" as the compound was not necessarily obtained by extraction. One might call is simply "Main compound". 
4. I guess your question is "What is an impurity?". Impurities is everything which differs from the main compound (or compounds in case of inseparable isomers).

Hope that helps.  

Best
